lib/rspec/core/example_group.rb

```diff
@@ -537,9 +537,15 @@ def self.store_before_context_ivars(example_group_instance)
         end
       end
 
+      # @deprecated use `RSpec.current_scope` instead
```

Comment:
```suggestion
```

Reply:
Its not necessary to deprecate this here

Reply:
Cherry-picked deprecation to https://github.com/rspec/rspec-core/pull/2880